ui/package.json

```diff
@@ -24,7 +24,6 @@
   "private": true,
   "dependencies": {
     "@clr/icons": "^0.11.12",
-    "@clr/ui": "^0.11.12",
```

Comment:
Looks like we can now remove our dependency on Clarity UI, thanks to all the work @evrii did!  (please confirm)

Reply:
We still have many Clarity Icons, so `@clr/icons` stays until https://precisionmedicineinitiative.atlassian.net/browse/RW-7683 is done

Reply:
Oh yeah, good call. Thanks for taking care of that. It seems to work just fine without it for me.